### DISCUSSION TITLE
Open to a PR fixing the example project on M1?

### DISCUSSION THREAD

**Discussion by naftalibeder** [rnmapbox/maps#1629] [Ideas]
I could do a PR that allows the `example` project to build properly on machines with the Apple M1 chip, but I just want to check if theres interest in it. It requires a couple of changes to the `Podfile` and iOS source.

As part of it, it would be nice to set up a `Gemfile` in `./ios`, so Cocoapods can be installed locally (in `./ios/vendor`) - in some environments it’s preferable not to have a global install like that.

Right now I just need to keep all this stuff unstaged locally which is… kind of annoying :)

**Comment by naftalibeder** [comment]
cc @ferdicus

**Reply by ferdicus** [reply]
sorry for the delay 😞 

Sure, sounds great actually - I could test compatibility with intel chip sets. 
We got quite some questions concerning build issue on M1 anyways. 

Thanks in advance

**Comment by naftalibeder** [comment]
Hey all, please see #1643 .
